### ISSUE TITLE
stitch doesn't work with network2

### ISSUE THREAD

**Issue by vincent766** [scikit-rf/scikit-rf#255]
stitch seems not to work with the new network2 format. 

The stitch (and other) method is missing for network2 class.

Vincent

**Comment by arsenovic** [comment]
network2 is just  a prototype, never completed.

**Comment by vincent766** [comment]
Hello Alex, 

I discovered your tool recently. I struggled with ADS linear simulator before. 

I found scikit-RF 100x better than ADS for all the non-standard calculation tasks. 
It works really well. There are no deadzones when combining two ideas for example. 



Thank you for the tool creation and maintenance. 

I will make some trials on my side for extending network2 and possibly submit it to github if it makes sense. 



Bye, 
Vincent 






----- Mail original -----

De: "alex arsenovic" <notifications@github.com> 
À: "scikit-rf/scikit-rf" <scikit-rf@noreply.github.com> 
Cc: "RFsketches" <vincent.fournier766@orange.fr>, "Author" <author@noreply.github.com> 
Envoyé: Lundi 22 Avril 2019 14:51:01 
Objet: Re: [scikit-rf/scikit-rf] stitch doesn't work with network2 (#255) 


network2 is just a prototype, never completed.
